brave/browser-laptop
# Adds clear/delete payments options

Resolves #8537 

## New options

Clear publishers option on shutdown
![image](https://user-images.githubusercontent.com/9574457/40040302-05f2b1a4-581a-11e8-9ae5-f62e9dc88ac6.png)


Manually clearing publishers and payment history
![image](https://user-images.githubusercontent.com/9574457/40040309-0bd825f4-581a-11e8-9136-d0a9a0320dc1.png)

While contribution is in progress
![image](https://user-images.githubusercontent.com/9574457/40643268-d8156ed2-631f-11e8-94cd-81835b126128.png)

Delete wallet
![image](https://user-images.githubusercontent.com/9574457/39990817-dfb7cfda-576d-11e8-9dc7-3edef11cd794.png)

Delete wallet confirmation
![image](https://user-images.githubusercontent.com/9574457/39990848-f2a490a6-576d-11e8-81f4-a118f91b3cef.png)


## Submitter Checklist:


## Test Plan:
1) make sure that synopsis is not reset when only clear `Browsing history` is selected when reseting browser
2) make sure that ledger table is cleared, but not payment history when `Brave Payments attention data` is selected when restarting browser
3) make sure that ledger table is not cleared when contribution is in progress
4) make sure that ledger table is cleared when clearing data manually `Brave Payments attention data`
5) make sure that payment history is cleared when clearing data manually `Brave Payments statements`
6) make sure that payment options are disabled when contribution is in progress when you are clearing data manually

## Reviewer Checklist:


Tests
